05_syntax_lab/01.py

```diff
@@ -1,5 +1,15 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Syntax excersize #1
+"""
+bigNum = 0
+number = 0
+

```

Comment:
The program works but may have a surprising result if all input numbers are negative. See if you can figure out what's wrong and fix it.